ThreeBotPackages/tfgrid_solutions/any_flist/chatflows/your_flist.py

```diff
@@ -0,0 +1,105 @@
+from Jumpscale import j
+import netaddr
+
+
+def chat(bot):
+    """
+    """
+
+    user_info = bot.user_info()
+    name = user_info["username"]
+    email = user_info["email"]
+    ips = ["IPV6", "IPV4"]
+    env = dict()
+
+    explorer = j.clients.threebot.explorer
+    if not email:
+        raise j.exceptions.BadRequest("Email shouldn't be empty")
+
+    form = bot.new_form()
+    flist = form.string_ask(
+        "Please add link of your flist to deploy it for example: https://hub.grid.tf/usr/example.flist"
+    )
+    pub_key = form.string_ask(
+        "Please add your public ssh-key (that will allow you to access the deployed container using ssh)"
+    )
+    env_vars = form.string_ask(
+        "Environment variables (optional. Comma-seperated env variables on container startup. For example: var1=value1, var=value2)"
```

Comment:
To set environment variables on your deployed container, enter comma-separated variable=value
For example: var1=value1, var2=value2.
Leave empty if not needed